rescoup/rc_m1/mod2/RC-01_MOD2_HIST.DATA

```diff
@@ -0,0 +1,210 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2024 Equinor 
+
+-- This deck is used for testing reservoir coupling. This one of 2 slave 
```

Comment:
Missing word I think.  "This one of 2" should **probably** be "This **is** one of 2".